### ISSUE TITLE
Remove plotdir

### ISSUE THREAD

**Issue by DanielAndreasen** [sousasag/ARES#4]
I have problems with [removal of plotdir](https://github.com/sousasag/ARES/blob/master/ARES_v2.c#L210-L214).

I'm getting the following error:

```
tar: plotdir/*.dat: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
tar: plotdir/*.png: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
```

The code runs, but it is annoying to look at the output. Is it possible to check if files exist before removing them? And should `plotdir` even be created when `plots_flag=0`?

**Comment by sousasag** [comment]
Done.

Forgot to fix that in the last change of ARES. Thanks.

**Comment by DanielAndreasen** [comment]
Thank you, it works. Fixed with commit c4d86eb3c88b (<-- some cool github stuff).
